middleware/requestid/requestid.go

```
@@ -36,16 +39,29 @@ func New(config ...Config) fiber.Handler {
+
 		// Continue stack
 		return c.Next()
 	}
 }
 
 // FromContext returns the request ID from context.
 // If there is no request ID, an empty string is returned.
-func FromContext(c fiber.Ctx) string {
-	if rid, ok := c.Locals(requestIDKey).(string); ok {
-		return rid
+func FromContext(c interface{}) string {
```

golangci-lint

[warning] use-any: since GO 1.18 'interface{}' can be replaced by 'any'

(revive)